### PR TITLE
Faster RGB & Depth noise models

### PR DESCRIPTION
**Pull request by mathfac**
## Motivation and Context
We noticed that RGB noise models are 7x times slower than whole Habitat step call.
@erikwijmans with @Skylion007  made our RGB noise models 50x faster with following PR.
- One minor change is to make GreedyFollower compatible with PyRobot actions. The change makes it possible to build shortest path on the fly with noise actuations.

## How Has This Been Tested
Existing [HSIM noise tests](https://github.com/facebookresearch/habitat-sim/blob/master/tests/test_sensors.py#L152-L207), [HAPI noise tests](https://github.com/facebookresearch/habitat-api/blob/master/test/test_sensors.py#L329).

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: habitat_sim/nav/greedy_geodesic_follower.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def __attrs_post_init__(self):
     def _find_action(self, name):
         candidates = list(
             filter(
-                lambda v: v[1].name == name,
+                lambda v: name in v[1].name,
```
Comment (Skylion007) [inline habitat_sim/nav/greedy_geodesic_follower.py]: Why is this change necessary?

Comment (erikwijmans) [inline habitat_sim/nav/greedy_geodesic_follower.py]: This is a little dangerous as things are assumed to be deterministic in the follower, but #546 will subsume this, so fine.

**Comment by mathfac** [comment]
Removed greedy_geodesic_follower change.
